lib/octokit/client/deployments.rb

```diff
@@ -1,69 +1,87 @@
+      end
+
+      # List deployment statuses
       #
-      # @param deployment_url [String] A URL for a deployment resource
+      # @param deployment_url [String] A URL for a deployment resource.
       # @return [Array<Sawyer::Resource>] A list of deployment statuses
       # @see https://developer.github.com/v3/repos/deployments/#list-deployment-statuses
       def deployment_statuses(deployment_url, options = {})
-        deployment = get(deployment_url, :accept => options[:accept])
+        deployment = get(deployment_url, accept: options[:accept])
         get(deployment.rels[:statuses].href, options)
       end
       alias :list_deployment_statuses :deployment_statuses
 
-      # Create a deployment status for a Deployment
+      # Create a deployment status
       #
-      # @param deployment_url [String] A URL for a deployment resource
-      # @param state [String] The state: pending, success, failure, error
-      # @option options [String] :target_url The target URL to associate with this status. Default: ""
-      # @option options [String] :description A short description of the status. Maximum length of 140 characters. Default: ""
-      # @return [Sawyer::Resource] A deployment status
+      # @param deployment_url [String] A URL for a deployment resource.
+      # @param state [String] The state of the status. Can be one of `error`, `failure`, `inactive`, `pending`, or `success`. **The `inactive` state requires a custom media type to be specified.**
+      # @param options [String] :target_url The target URL to associate with this status. This URL should contain output to keep the user updated while the task is running or serve as historical information for what happened in the deployment.
+      # @param options [String] :log_url This is functionally equivalent to `target_url`. We will continue accept `target_url` to support legacy uses, but we recommend modifying this to the new name to avoid confusion. **This parameter requires a custom media type to be specified.**
+      # @param options [String] :description A short description of the status. Maximum length of 140 characters.
+      # @param options [String] :environment_url Sets the URL for accessing your environment. **This parameter requires a custom media type to be specified.**
+      # @param options [Boolean] :auto_inactive Adds a new `inactive` status to all non-transient, non-production environment deployments with the same repository and environment name as the created status's deployment. **This parameter requires a custom media type to be specified.**
+      # @return <Sawyer::Resource> The new deployment status
       # @see https://developer.github.com/v3/repos/deployments/#create-a-deployment-status
       def create_deployment_status(deployment_url, state, options = {})
-        deployment = get(deployment_url, :accept => options[:accept])
+        deployment = get(deployment_url, accept: options[:accept])
         options[:state] = state.to_s.downcase
```

Comment:
Enums are going to be interesting. For now I assume that they should be normalized by downcasing them. This might not actually be true.